### PR TITLE
Fix grammatical error

### PR DESCRIPTION
**Pull request by Jalkhov**
Fix grammatical error in docs

### CODE REVIEW

File: src/flask/scaffold.py [hunk 1]
```diff
@@ -589,8 +589,8 @@ def teardown_request(self, f: TeardownCallable) -> TeardownCallable:
         stack of active contexts.  This becomes relevant if you are using
         such constructs in tests.
 
-        Teardown functions must avoid raising exceptions, since they . If they
-        execute code that might fail they
+        Teardown functions must avoid raising exceptions, because if
```
Comment (davidism) [inline src/flask/scaffold.py:592]: Sounds better with a stop: "...raising exceptions. If they..."

File: src/flask/scaffold.py [hunk 2]
```diff
@@ -589,7 +589,7 @@ def teardown_request(self, f: TeardownCallable) -> TeardownCallable:
         stack of active contexts.  This becomes relevant if you are using
         such constructs in tests.
 
-        Teardown functions must avoid raising exceptions, because if
+        Teardown functions must avoid raising exceptions. If
         they execute code that might fail they
         will have to surround the execution of these code by try/except
```
Comment (davidism) [inline src/flask/scaffold.py:594]: "that code" instead of "these code"

File: src/flask/scaffold.py [hunk 3]
```diff
@@ -589,7 +589,7 @@ def teardown_request(self, f: TeardownCallable) -> TeardownCallable:
...
         statements and log occurring errors.
```
Comment (davidism) [inline src/flask/scaffold.py:595]: "any errors" instead of "occurring errors"

File: src/flask/scaffold.py (hunk 1 above)
Comment (Jalkhov) [reply src/flask/scaffold.py]: Ready

**Pull request merged**